packages/oauth-helper/src/index.ts

```diff
@@ -0,0 +1,468 @@
+
+// TODO: Not using RequestListner, should I?
```

Comment:
Do I need RequestListner for any reason? It was in the original spec but I didn't end up using it